alcove/query/api.py

```
@@ -19,8 +29,136 @@
+
+if _METRICS_ENABLED:
+    _start_time = time.monotonic()
+    _total_requests = 0
+    _active_requests = 0
+    _error_count_4xx = 0
+    _error_count_5xx = 0
+    _largest_view_chars = 0
+    _recent_times: _collections.deque = _collections.deque(maxlen=100)
+    _access_log_file = None
+
+    def _get_access_log():
```

Ruff (0.15.5)

[warning] Missing return type annotation for private function `_get_access_log`

(ANN202)